dapps/marketplace/src/components/_UserProfileCreated.js

```diff
@@ -35,22 +35,11 @@ const UserProfileCreated = ({ onCompleted, isMobile, referralReward }) => {
         {referralReward ? (
           <>
             <Link
-              to="/welcome"
+              to="/campaigns"
```

Comment:
I thought @micahalcorn only wanted this conditional, which is why the sessions tore `getStartedRedirect` was used.  If this is cool with Micah, I don't care.

Reply:
@mikeshultz what is the condition? Under no circumstance that I'm aware of should a member of Origin Rewards see the /welcome page.

Reply:
Appears to be fine with @micahalcorn.  Resolved.